gems/activerecord/6.1/_test/Steepfile

```diff
@@ -0,0 +1,25 @@
+target :test do
+  signature "."
+  check "."
+
+  repo_path "../../../"
+
+  library "pathname"
+  library "logger"
+  library "mutex_m"
+  library "date"
+  library "monitor"
+  library "singleton"
+  library "tsort"
+  library "rack"
+
+  library "activerecord:6.1.4.1"
+  library "activesupport:6.1.4.1"
+  library "actionpack:6.1.4.1"
+  library "activejob:6.1.4.1"
+  library "activemodel:6.1.4.1"
+  library "actionview:6.1.4.1"
+  library "railties:6.1.4.1"
+
+  typing_options :strict
```

Comment:
Could you use the new configuration of Steep which is introduced v0.46.0? For example: https://github.com/ruby/gem_rbs_collection/pull/58


The other parts of this PR looks good :+1: